dcos/marathon.py

```diff
@@ -290,7 +290,10 @@ def add_app(self, app_resource):
         else:
             app_json = app_resource
 
-        response = _http_req(http.post, url,
+        if not isinstance(app_json, list):
+            app_json = [app_json]
+
+        response = _http_req(http.put, url,
```

Comment:
Makes sense. The app definition returned should contain the deployment idea, right? Could we parse it out so we can continue to return "Created deployment x"?